README.md

```diff
@@ -85,19 +127,22 @@ If you do not want to share storage with Sandstorm, you can omit the
 In addition to the files used by sandstorm, `tempest` will create a
 couple extra things underneath that path, namely:
 
-- an extra directory at `sandstorm/mnt`
-- a sqlite3 database at `sandstorm/sandstorm.sqlite3`
+- a sqlite3 database at `/opt/sandstorm/var/sandstorm/sandstorm.sqlite3`
+  (that's it for now, we will try to update as things progress)
```

Comment:
I didn't see a `mnt` anywhere but maybe it's because I didn't get far enough in running tempest.